antarest/study/storage/variantstudy/model/command/create_cluster.py

```diff
@@ -53,10 +53,17 @@ class CreateCluster(ICommand):
 
     @field_validator("cluster_name", mode="before")
     def validate_cluster_name(cls, val: str) -> str:
```

Comment:
I think we can create an annotated type that we can reuse everywhere, instead of duplicating the method in all classes.

https://docs.pydantic.dev/latest/concepts/validators/#annotated-validators